package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@pedroslopez/moduleraid": "^5.0.2",
     "fluent-ffmpeg": "^2.1.2",
+    "jimp": "^0.16.1",
```

Comment:
There is a reason to choose jimp instead of sharp?
Sharp is way faster: https://www.peterbe.com/plog/sharp-vs-jimp